indico/web/client/js/react/components/pictures/PictureCropper.jsx

```diff
@@ -0,0 +1,86 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Cropper from 'react-cropper';
+import {Header, Icon, Button} from 'semantic-ui-react';
+
+import 'cropperjs/dist/cropper.css';
+import './Picture.module.scss';
+
+export function PictureCropper({cropperRef, src, onCrop, backAction, minCropSize}) {
+  let size = Math.max(minCropSize, 500);
+  size += 200;
```

Comment:
```suggestion
  const size = Math.max(minCropSize, 500) + 200;
```

also, a comment on why these numbers are used might be useful...

Reply:
The default minsize of 500 is to allow the field fit well inside the registration form. And the 200 is to offset the min values set by the users. So the total default size is 700. For example, if a minSize of 700 is set in the field settings then the picture field is 900 so that there's enough area to crop or select the image.